src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -230,114 +223,109 @@ public String toString() {
 	 * <li>When reloading all scripts, scripts that were removed are disabled
 	 * after everything has been reloaded
 	 * <li>Script infos returned by most methods are inaccurate
+	 *
 	 * @return If main thread is not blocked when loading.
 	 */
 	public static boolean isAsync() {
 		return loadAsync;
 	}
-	
+
 	/**
 	 * All loaded script files.
 	 */
 	@SuppressWarnings("null")
-	static final Set<File> loadedFiles = Collections.synchronizedSet(new HashSet<>());
-	
+	private static final Set<File> loadedFiles = Collections.synchronizedSet(new HashSet<>());
+
 	@SuppressWarnings("null") // Collections methods don't return nulls, ever
 	public static Collection<File> getLoadedFiles() {
 		return Collections.unmodifiableCollection(loadedFiles);
 	}
-	
+
 	// Initialize and start load thread
 	static {
 		loaderThread = new AsyncLoaderThread();
 		loaderThread.start();
 	}
-	
-	private static class AsyncLoaderThread extends Thread {
-		
-		public AsyncLoaderThread() { }
 
+	private static class AsyncLoaderThread extends Thread {
 		@Override
 		public void run() {
 			while (true) {
 				try {
 					loadQueue.take().run();
 				} catch (InterruptedException e) {
+					//noinspection ThrowableNotThrown

```

Comment:
IDE-specific warning suppression comments are not necessary. Please just configure your IDE to not create the warnings, if they are not for a reason. But chances are that the warnings _do_ have a reason, in which case that should be fixed.

Reply:
I don't think there is any reason. However, changing signatures of public methods can produce endless amounts of _fun_ when people use addons...

Reply:
Oh yeah right, I guess it's one of the top-used methods by addons (in the init methods at least), didn't think about that